src/pages/explorer/index.tsx

```diff
@@ -9,9 +9,27 @@ import { StrategyProvider, useStrategyCtx } from 'hooks/useStrategies';
 import { ExplorerTabs } from 'components/explorer/ExplorerTabs';
 import { useEffect, useState } from 'react';
 import { explorerEvents } from 'services/events/explorerEvents';
+import { toPairSlug } from 'utils/pairSearch';
+import config from 'config';
 
 export const ExplorerPage = () => {
   const { slug, type } = useExplorerParams();
+  const navigate = useNavigate();
```

Comment:
Since you know the page you're on you can do that : 
```suggestion
  const navigate = useNavigate({ from: '/explore/$type/$slug' });
```